### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by DenisaRohunova**
Please check my code.

### CODE REVIEW

File: Pet-Clinic-Frontend/src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-
-
   return (
     <>
-    <h1>Main page</h1>
-  
+     <img src={clinicImage} alt="Clinic" className="clinic-image" /> 
+      <h1 className="h1">Pet Clinic</h1>
+      <h2 className="h2">Welcome to our fabulous clinic</h2>
+
```
Comment (Normelik) [inline Pet-Clinic-Frontend/src/App.tsx:10]: try to put it as a separate page into the pages folder, than call it here (like <LandingPage />)

Comment (DenisaRohunova) [reply Pet-Clinic-Frontend/src/App.tsx]: Done :))

File: Pet-Clinic-Frontend/src/pages/Landing_page.tsx
Comment (Normelik) [inline Pet-Clinic-Frontend/src/pages/Landing_page.tsx:1]: you can simplify the name, just Landing. It's in the folder name "pages" so everyone knows it's a page. But of course it's up to you.

File: Pet-Clinic-Frontend/src/pages/Landing_page.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import './Landing_page.css';
+import clinicImage from "../images/petclinic.png";
+
+function App() {
```
Comment (Normelik) [inline Pet-Clinic-Frontend/src/pages/Landing_page.tsx:4]: perhaps a more descriptive function name? On the other hand, this also works

**Review by Normelik — COMMENTED** [review]
The page is awesame. I like it. You can improve the code a bit, but you don't have to.

**Comment by shruska** [comment]
Nice paw and images. There should be "The landing page should contain a catchy description of the application."  But I think for now it is perfect.

File: Pet-Clinic-Frontend/src/pages/Landing_page.tsx (hunk 2 above)
Comment (shruska) [reply Pet-Clinic-Frontend/src/pages/Landing_page.tsx:4]: Yeah maybe LandingPage?

**Review by ReneVajda1 — COMMENTED** [review]
beautiful

File: Pet-Clinic-Frontend/package.json [hunk 3]
```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.23.1" 
   },
```
Comment (markkovari) [inline Pet-Clinic-Frontend/package.json:16]: Nice

File: .vscode/launch.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+"configurations": [
+{
+"type": "chrome",
+"request": "launch",
+"name": "Launch Chrome against localhost",
+"url": "http://localhost:8080",
+"webRoot": "${workspaceFolder}"
+}
+]
+}
```
Comment (markkovari) [inline .vscode/launch.json:15]: Can we format this code?

**Review by markkovari — CHANGES_REQUESTED** [review]

**Review by ReneVajda1 — COMMENTED** [review]
reviewing again

**Review by markkovari — APPROVED** [review]
image

**Review by Normelik — APPROVED** [review]
lets merge it :)

**Pull request merged**